dart/lib/src/sentry_isolate.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:meta/meta.dart';
+
+import '../sentry.dart';
+
+/// Record isolate errors with the Sentry SDK.
+extension SentryIsolate on Isolate {
+
+  /// Calls [addErrorListener] with an error listener from the Sentry SDK. Store
+  /// the returned [RawReceivePort] if you want to remove the Sentry listener
+  /// again.
+  ///
+  /// Since isolates run concurrently, it's possible for it to exit before the
+  /// error listener is established. To avoid this, start the isolate paused,
+  /// add the listener and then resume the isolate.
+  RawReceivePort addSentryErrorListener() {
+    final hub = Sentry.currentHub;
+    final options = hub.options;
+
+    final port = _createPort(hub, options);
+    addErrorListener(port.sendPort);
+    return port;
+  }
+
+  /// Pass the [receivePort] returned from [addSentryErrorListener] to remove
+  /// the sentry error listener.
+  void removeSentryErrorListener(RawReceivePort receivePort) {
+    receivePort.close();
```

Comment:
I believe we should not call `close` but` rather only remove the listener, the method is called `removeSentryErrorListener`

Reply:
We did do this in the integration, it's ok to change this behaviour?

Reply:
The integration should call `close` in the `receivePort`, but not the extension itself.
The integration is alive during app's lifecycle.
The extension can be used specifically for a few things.